### DISCUSSION TITLE
Expo build:ios Forbidden

### DISCUSSION THREAD

**Discussion by anujdivesh** [expo/expo#14471] [Archived: Questions] [answered]
Hi,
I was trying to run expo build:ios on my mac to deploy my app to apple dev account. I get the following error: 
**Network timeout 'AccessForbidden Error'
Apple 403 detected - Access Forbidden**

Any leads will be appreciated.

**Comment by brentvatne** [comment, marked as answer]
possibly apple's servers were not working properly at the time, are you still having this issue?

**Comment by anujdivesh** [comment]
It worked Later. I am not sure what was the issue. Thanks :)
